lib/python/src/aria/ops/timer.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import annotations
+
+from logging import Logger
+from time import time
```

Comment:
when you do this, it becomes difficult to use other `time` methods (like `time.sleep()`)

Reply:
It shouldn't be too difficult, just add `from time import sleep` and then call `sleep(...)`. Also, the imports will only affect this file, so it won't cause anything to work different for calling code. If we start needing more methods from the `time` module for the Timer class we can re-evaluate then.